packages/css/src/components/image/README.md

```diff
@@ -1,23 +1,22 @@
 # Image
 
-Toont een afbeelding.
+Displays an image.
 
-## Richtlijnen
+## Guidelines
 
-- Vergeet niet om een beschrijving van de afbeelding op te nemen in het `alt`-attribuut.
-  Dit zorgt ervoor dat gebruikers van schermlezers deze informatie ook tot zich kunnen nemen.
-  Daarnaast kan het helpen bij zoekmachineoptimalisatie.
-  Alleen voor decoratieve afbeeldingen is zo’n beschrijving niet nodig. Gebruik in dit geval `alt=""`.
-  Denk aan afbeeldingen die weinig toevoegen aan de nabije tekst of afbeeldingen die louter bijdragen aan het ontwerp of de sfeer van de pagina (bron: [W3C Web Accessibility Initiative](https://www.w3.org/WAI/tutorials/images/decorative/)).
-- Specificeer eventueel meerdere kandidaten voor de afbeelding via het `srcSet`-attribuut.
-  Bijvoorbeeld een kleine, middelgrote en grote variant voor de diverse schermformaten.
-  Zo voorkom je het onnodige downloaden van grote bestanden.
-  Doe dit vooral voor de hoofdafbeelding van een pagina,
-  daar is het verschil tussen de formaten op een smal en breed scherm het grootst.
-- Zorg ervoor dat de afbeelding een beeldverhouding heeft die ondersteund wordt door het [Aspect Ratio](?path=/docs/layout-aspect-ratio--docs) component.
+- Do not forget to include a description of the image in the `alt` attribute.
+  This ensures that screen reader users can also access this information.
+  Additionally, it can aid in search engine optimization.
+- A description is unnecessary for decorative images; use `alt=""` for these.
+  Examples are images that add little to the nearby text or pictures that purely contribute to the design or atmosphere of the page (source: [W3C Web Accessibility Initiative](https://www.w3.org/WAI/tutorials/images/decorative/)).
+- Optionally specify multiple candidates for the image through the `srcSet` attribute.
```

Comment:
Now that I read this (...) we need an example using srcset.

Reply:
`Overlap` has one; feel free to open a PR adding an example to `Image`.